docs/config_extra.md

```diff
@@ -0,0 +1,48 @@
+# Additional Device Specific Configuration Settings
+These settings can be set using the following format:
+```yaml
+insteon:
+  devices:
+    switch:
+      - aa.bb.cc: lamp1
+        extra_config: 1
+      - 11.22.33: lamp2
+```
+In this case, `lamp1` would have the value of 1 set for the `extra_config` setting.  This setting __only__ affects `lamp1` and will not affect `lamp2`.
```

Comment:
I get the impression that "extra_config" isn't actually a real parameter that can be configured.  Is there a way to describe the things in this section (i.e. names are required, how to indent, don't use dashes) without relying on a fake parameter name?  I wonder if some users might feel like they have to literally add "extra_config: 1" to their config in order to use other settings described in this file?

Generally speaking, this file seems structured for describing multiple types of extra-config options, but at least for now, there's only one that's possible (and an advanced one, at that).  Maybe until others are available, it would be better to make this file only talk about how to use "min_hops" and to start out with the "Advanced Settings" disclaimer.  It can always be re-formatted to look more like it is now once other non-advanced and/or device-specific settings are added.

Reply:
By your logic, a user might conclude that they need to enter `min_hops` to enable extra configuration settings.  I would rather the example be something that will not cause issues.  They only need to look half a page down to see the `min_hops` setting.

Reply:
Ok, lets leave this as is.